x-pack/test_serverless/functional/test_suites/observability/cases/create_case_form.ts

```diff
@@ -64,75 +64,5 @@ export default ({ getService, getPageObject }: FtrProviderContext) => {
       const button = await find.byCssSelector('[data-test-subj*="case-callout"] button');
       expect(await button.getVisibleText()).equal('Add connector');
     });
-
-    it('displays errors correctly while creating a case', async () => {
```

Comment:
I think we have a test for it in ESS, right?

Reply:
What is ESS 😄 ?

Reply:
Ahh I already removed test from it: `x-pack/test/functional_with_es_ssl/apps/cases/group1/create_case_form.ts`